src/app/api/api_v1/endpoints/detections.py

```diff
@@ -112,10 +116,36 @@ async def fetch_detections(
     if UserRole.ADMIN in token_payload.scopes:
         return [elt for elt in await detections.fetch_all()]
 
-    cameras_list = await cameras.fetch_all(("organization_id", token_payload.organization_id))
+    cameras_list = await cameras.fetch_all(filter_pair=("organization_id", token_payload.organization_id))
     camera_ids = [camera.id for camera in cameras_list]
 
-    return await detections.get_in(camera_ids, "camera_id")
+    return await detections.fetch_all(in_pair=("camera_id", camera_ids))
+
+
+@router.get("/unacknowledged/from", status_code=status.HTTP_200_OK, summary="Fetch all the unacknowledged detections")
+async def fetch_unacknowledged_detections(
+    from_date: Optional[datetime] = None,
```

Comment:
Path, payload or Form data, but I'd suggest JSON Payload to get proper pydantic validation & for the sake of uniformity across the routes

Reply:
hum Form isn't  equal to Payload ? I see that we are already using Form but I don't see any Payload

Reply:
I mean we define a Pydantic inherited class with one datetime attribute called "starting_date" or "from_date". So that we'll get payload validation.

Form is what we use here for user login for instance